tools/gui/config.json

```diff
@@ -0,0 +1,29 @@
+{
+    "chains": {
```

Comment:
We should probably update the config.json to match [this](https://github.com/Lay3rLabs/avs-toolkit/blob/main/tools/cli/config.json)

Reply:
Even better, we should use the same config.json for cli and gui

Reply:
good call, consolidated into one config file in https://github.com/Lay3rLabs/avs-toolkit/pull/74/commits/3fcff82696b63897dc83e85cc9c7f0543e74faf3